lms/tasks/email_digests.py

```diff
@@ -59,7 +58,7 @@ def send_instructor_email_digest_tasks():
                     # Note here that we are considering earlier launches
                     # We rather take a few more courses that miss some cases
                     # where the launch that originated the annotations was made around the cutoff time.
-                    Event.timestamp >= created_before - timedelta(days=14),
+                    Event.timestamp >= created_after - timedelta(days=7),
```

Comment:
Just for good measure, I'm re-setting this time range in which we look for events as well. This will reduce the number of `h_userid`'s that the task processes and so reduce the name of `send_instructor_email_digest()` tasks that we run per day. I don't think this is necessary, but just in case